### PR TITLE
Release 4.6.1

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**
I created this [support PR](https://github.com/magpiemodel/magpie/pull/491) to just show the differences between the current develop and new master, might be easier to look at than this massive PR with 78 changed files.

### CODE REVIEW

File: modules/10_land/landmatrix_dec18/presolve.gms [hunk 1]
```diff
@@ -10,25 +10,15 @@
 *' @code Some of the land use transitions are restricted:
 
 *' No planted forest on natveg areas
-v10_lu_transitions.fx(j,"primforest","forestry") = 0;
-*v10_lu_transitions.fx(j,"secdforest","forestry") = 0;
-*v10_lu_transitions.fx(j,"other","forestry") = 0;
+vm_lu_transitions.fx(j,"primforest","forestry") = 0;
```
Comment (pvjeetze) [inline modules/10_land/landmatrix_dec18/presolve.gms:13]: @k4rst3ns: Why were these constraints for forestry on secdforest and other removed?

**Review by pvjeetze — COMMENTED** [review]
I added a question @k4rst3ns, but otherwise all looks fine.

**Review by tscheypidi — APPROVED** [review]

**Review by merfort — APPROVED** [review]
Not an expert on the details, but looks good to me.

**Review by tscheypidi — APPROVED** [review]
looks good

File: modules/10_land/landmatrix_dec18/presolve.gms (hunk 1 above)
Comment (k4rst3ns) [reply modules/10_land/landmatrix_dec18/presolve.gms:13]: They have been removed as they have been commented out for almost 2 years.

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ The Coding Etiquette explains also the naming conventions and other
 structural characteristics.
 
 ## COPYRIGHT
-Copyright 2008-2021 Potsdam Institute for Climate Impact Research (PIK)
+Copyright 2008-2022 Potsdam Institute for Climate Impact Research (PIK)
```
Comment (k4rst3ns) [inline README.md:46]: Funny that you changed it here.
All code files of magpie itself have a header that still states:
"*** | (C) 2008-2021 Potsdam Institute for Climate Impact Research (PIK) [...]"

But I already talked to Jan, that we do the header update for the first release in 2023 ^^'

**Review by k4rst3ns — APPROVED** [review]
Looks good to me.

**Pull request merged**